packages/vsf-storyblok-module/components/global/Img.vue

```diff
@@ -54,7 +57,7 @@ export default {
   props: {
     detectWebp: {
       type: Boolean,
-      default: true
+      default: get(config, 'storyblok.imageService.defaultWebp', true)
```

Comment:
@benjick @indiebytes Also I would argue to keep the default value of `detectWebp` as `false`. Partly because the feature is not 100% working properly and also because it would mean an unexpected change when upgrading this module if it was default `true`.
WDYT?

Reply:
I kept it as `true` here, because I didn't want to change it to `false` without getting your approval.